src/components/work/BibliographicData/ManifestationFull.js

```diff
@@ -208,6 +244,16 @@ export default function wrap({ manifestation, work, workId, localizations }) {
     });
   };
 
+  const openOrderModal = (pid) => {
+    modal.push("order", {
+      title: Translate({ context: "modal", label: "title-order" }),
+      pid,
+      workId,
+      type: manifestation.materialType,
+      orderType: "singleManifestation",
```

Comment:
Kunne også bære være singleManifestation: true, så vi ikke behøver oversætte fra string to boolean i wrap

Reply:
jo men jeg tænkte at der måske ville komme andre 'orderType'